docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md

```diff
@@ -9,17 +9,28 @@ page](https://mxnet.apache.org/get_started/download).
 
 Run the following command:
 
-<div class="v1-6-0">
+<div class="v1-7-0">
 {% highlight bash %}
 pip install mxnet
 {% endhighlight %}
 
+Start from 1.7.0 release, MKL-DNN is enabled in pip packages by default. Which are
+optimized for Intel hardware. You can find performance numbers
+in the <a href="https://mxnet.io/api/faq/perf#intel-cpu">MXNet tuning guide</a>.
```

Comment:
Link to mxnet.apache.org/

Reply:
The perf link under mxnet.apache.org will contain the version number like "https://mxnet.apache.org/versions/1.6/api/faq/perf.html#intel-cpu". Is there any basic link without version number which can redirect to the latest version of the page? 
Please check my latest update.